ESISharp/Web/EsiAuthRequest.cs

```diff
@@ -26,25 +26,29 @@ internal EsiAuthRequest(ESIEve SwaggerObject, string RequestPath)
 
         private bool VerifyCredentials()
         {
-            if (AuthenticatedEasyObject.SSO.AuthToken == null && AuthenticatedEasyObject.SSO.ImplicitToken == null)
+            lock(this)
```

Comment:
I guess I can understand the property part, but why static? It only needs to lock for the specific SSO instance, not globally.